server/routes/teacherRegistration.js

```diff
@@ -0,0 +1,51 @@
+import express, { json } from 'express';
+import db from "../db/connection.js";
+import User from "../db/schemas.js";
+
+const router = express.Router();
+router.use(express.json());
+router.use(express.urlencoded({extended: true})); 
+
+
+async function validateId(user_id) {
+    try {
+      const user = await User.findById(user_id);
+      return user ? false : true; 
+    } catch (error) {
+      console.error(error);
+      return false; 
+    }
+  }
+
+function validatePassword(pw){
+    return true;
```

Comment:
please don't forget to implement this function! we should probably look at check for a certain length, if it contains alphanumeric characters as well as special symbols